### PR TITLE
Updated cfadmin with link and examples

### PR DESCRIPTION
**Pull request by gratzc**

### CODE REVIEW

File: data/en/cfadmin.json [hunk 1]
```diff
@@ -13,13 +13,953 @@
...
+			    action=\"collection\"\n
+			    type=\"web|server\"\n
+			    password=\"password\"\n
+			    collectionAction=\"create\"\n
+			    collection=\"#form.collName#\"\n
+			    path=\"#form.collPath#\"\n
+			    language=\"#form.collLanguage#\"\n
+			    remoteClients=\"arrayOfClients\">\n
+
+			\n<cfadmin\n
```
Comment (shaedrich) [inline data/en/cfadmin.json]: Could you split each example into an own structure inside the `examples` array?

File: data/en/cfadmin.json [hunk 2]
```diff
@@ -13,13 +13,953 @@
 		"lucee": {"minimum_version":"", "notes":"", "docs":"http://docs.lucee.org/"}
 	},
 	"links": [
+		{
+			"title":"Lucee Documentation",
+			"url":"http://docs.lucee.org/guides/Various/cfadmin.html",
+			"description":"Lucee documentation for cfadmin tag."
```
Comment (shaedrich) [inline data/en/cfadmin.json]: Put this under `engines.lucee.docs`

**Comment by gratzc** [comment]
Updated with suggestions

File: data/en/cfadmin.json [hunk 3]
```diff
@@ -36,326 +36,473 @@
 			    collection=\"#form.collName#\"\n
 			    path=\"#form.collPath#\"\n
 			    language=\"#form.collLanguage#\"\n
-			    remoteClients=\"arrayOfClients\">\n
-
-			\n<cfadmin\n
+			    remoteClients=\"arrayOfClients\">
```
Comment (shaedrich) [inline data/en/cfadmin.json]: Now you missed the closing quote after each example.

File: data/en/cfadmin.json [hunk 4]
```diff
@@ -26,8 +26,8 @@
 	],
 	"examples": [
 		{
-			"title":"Below you will find all the actions sorted alphabetically embedded in examples for them:",
-			"description":"These are examples provided by Lucee for the cfadmin actions in alphabetical order",
+			"title":"Create an admin collection",
+			"description":"",
 			"code":"<cfadmin\n
 			    action=\"collection\"\n
```
Comment (shaedrich) [inline data/en/cfadmin.json]: Unfortunately JSON does not support visual multiline strings. Only `\n` work and works properly in single-line string when you output it.

Comment (gratzc) [reply data/en/cfadmin.json]: yes i know i am working on it

File: data/en/cfadmin.json [hunk 5]
```diff
@@ -1,26 +1,619 @@
...
+      "description": "",
+      "code": "<cfadmin\n action=\"collection\"\n type=\"web|server\"\n password=\"password\"\n collectionAction=\"create\"\n collection=\"#form.collName#\"\n path=\"#form.collPath#\"\n language=\"#form.collLanguage#\"\n remoteClients=\"arrayOfClients\">"
+    },
+    {
+      "title": "Delete an admin collection",
+      "description": "",
+      "code": "<cfadmin\n action=\"collection\"\n type=\"web|server\"\n password=\"password\"\n collectionAction=\"delete\"\n collection=\"#form.name[key]#\"\n remoteClients=\"arrayOfClients\">"
+    },
+    {
+      "title": "",
```
Comment (shaedrich) [inline data/en/cfadmin.json]: Our tests fail if a title is empty. You can move the description to the title instead.

**Comment by pfreitag** [comment]
Thanks @gratzc

**Pull request merged**